litcal_calendar_cache.py

```
@@ -0,0 +1,178 @@
+    calendar_id: str
+    year: int
+    locale: str = "en"
+
+    def __post_init__(self):
+        """Validate the calendar type."""
+        valid_types = ("general", "national", "diocesan")
+        if self.calendar_type not in valid_types:
+            raise ValueError(
+                f"Invalid calendar type: {self.calendar_type}. "
+                f"Must be one of {valid_types}"
+            )
```

Ruff (0.14.3)

Avoid specifying long messages outside the exception class

(TRY003)